riscv/encoding.h

```diff
@@ -172,6 +172,8 @@
 #define MSTATEEN0_FCSR     0x00000002
 #define MSTATEEN0_JVT      0x00000004
 #define MSTATEEN0_HCONTEXT 0x0200000000000000
+#define MSTATEEN0_SCSRIND  0x1000000000000000
+#define HSTATEEN0_SCSRIND  0x1000000000000000
```

Comment:
We should not be hand-editing `encoding.h`. It needs to be generated from the riscv-opcodes repo, and any change to it here should update the comment at the top that explains which git hash it was generated from.

Reply:
Agree -- this was expedient and intended to ensure that it compiles :).

Reply:
Dropped -- see https://github.com/riscv/riscv-opcodes/pull/178 for the relevant changes.